services/web/server/src/simcore_service_webserver/socketio/handlers.py

```diff
@@ -93,7 +93,7 @@ async def set_user_in_rooms(
 ) -> None:
     user_id = request.get(RQT_USERID_KEY, ANONYMOUS_USER_ID)
     primary_group, user_groups, all_group = await list_user_groups(app, user_id)
-    groups = [primary_group] + user_groups + [all_group]
+    groups = [primary_group] + user_groups + ([all_group] if bool(all_group) else [])
```

Comment:
@sanderegg This caused an error if the **all_group** was missing. Found out about this while I had the db in an inconsistent state. Should I leave this in or remove it?

Reply:
The issue is that at that point I had no error in the logs (I don't know why). I caught the bug with the debugger.

Reply:
well the point is. if the all group is missing it means there is a problem. so at least log an error.

Reply:
If I revert the change, no error will be raised and I don't know why.